com.jetbrains.IntelliJ-IDEA-Ultimate.json

```diff
@@ -21,6 +21,7 @@
     "--socket=pulseaudio",
     "--socket=ssh-auth",
     "--socket=x11",
+    "--env=JAVA_HOME=/app/extra/idea-IU/jbr",
```

Comment:
What change triggered the need to specify this one?

I think `/app/extra/idea-IU/jb` refers to a JRE rather than an SDK. It may be all right for running IDEA, but my guess is that everything-Java will try to use `/app/extra/idea-IU/jb` unless `JAVA_HOME` is overwritten.

Reply:
jbr is the only one jdk included in package, if you want to use openjdk from extension just override it, no ?

in this way idea runs out of the box